app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -2,7 +2,14 @@ module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Configurati
   extend ActiveSupport::Concern
 
   module ClassMethods
+    def provider_params(params)
+      authentication_id = params.delete('authentication_id') || params.delete(:authentication_id)
```

Comment:
The `string` form is removed now since API team has merged the patch to always convert keys to `symbol`.